index.html

```diff
@@ -4,36 +4,115 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width,initial-scale=1.0" />
     <title>TripHouse</title>
+    <link rel="icon" href="./images/favicons/favicon.ico" />
     <link rel="stylesheet" href="styles/style.css" />
   </head>
   <body class="body">
+    <div class="top-section">
+      <div class="img-top">
+        <div class="header">
+          <div class="header-logo-left"></div>
+
+          <div class="header-logo-right">
```

Comment:
here you can call header-nav -  becouse in future it will be nav bar and contain links and settings